src/Database/LSMTree/Internal/Run/BloomFilter.hs

```diff
@@ -22,4 +24,66 @@
 -- The above list is a sketch. Functionality may move around, and the list is
 -- not exhaustive.
 --
-module Database.LSMTree.Internal.Run.BloomFilter () where
+module Database.LSMTree.Internal.Run.BloomFilter (
+    -- * Tuning a la Monkey
+    monkeyFPR
+  , monkeyBits
+  , monkeyHashFuncs
+    -- * Bloom
+  , Bloom
+  , Bloom.elem
+    -- ** Hashes
+  , Hash
+  , Hash.Hashable (..)
+  , Hash.cheapHashes
```

Comment:
I don't think we can use these. As `bloomfilter` package *does not* provide any serialization guarantees, i.e. is a memory only package, it for sure doesn't provide strong guarantees that the `cheapHashes` will have the same implementation indefinitely.

Reply:
Alright, maybe we can "fork" the implementation of `cheapHashes` so we don't have to come up with our own hashing functions. How does that sound?

Reply:
Or, we can add a number of golden tests so we can see when the hash function changes

Reply:
The file format will still depend on an implementation detail of a package, i.e. cannot be stated independently in a specification (of said file format).

If that is not an issue, it's ok.

I wouldn't use a persistence layer which has such 3rd party implementation dependency myself.